lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1393,6 +1439,41 @@ def __get_vnic_profile_id(self, nic):
                 )
             )
 
+    def __attach_numa_nodes(self, entity):
+        numa_nodes_service = self._service.service(entity.id).numa_nodes_service()
+
+        if len(self.param('numa_nodes')) > 0:
+            # Remove all existing virtual numa nodes before adding new ones
+            existed_numa_nodes = numa_nodes_service.list()
+            existed_numa_nodes.sort(reverse=len(existed_numa_nodes) > 1 and existed_numa_nodes[1].index > existed_numa_nodes[0].index)
+            for current_numa_node in existed_numa_nodes:
+                numa_nodes_service.node_service(current_numa_node.id).remove()
```

Comment:
We support `update`, would it be better (safer) to use it, instead of remove/create it?

Reply:
What do you mean by "update" and why it is safer?
Since AFAIK there is no meaning to add/delete a specific single NUMA node and ignore all others (i.e. each time you change NUMA settings - you should edit all nodes all together) then I think is is preferable to remove+add all numa nodes each time you change something. 
You can also remove all NUMA nodes all together of course.

Reply:
** If you enable watchdog

Reply:
Thanks OK, I just wasn't sure how it's working.

Reply:
Once Martin will review.